en/style-guide.adoc

```diff
@@ -167,7 +167,7 @@ Incorrect
  blue-fill: func [...		;-- should be fill-blue
 ----
 
-There is an exception to those naming rules which applies to OS or non-Red third-party API names. In order to make API-specific function and structures field names easily recognizable, their original name should be used. It visually helps distinguish such imported names from regular Red or Red/System code. For example:
+There is an exception to naming rules which apply to OS or non-Red third-party API names. To make API-specific function- and structure field names easy to recognize, their original name should be used. It visually helps to distinguish imported names from regular Red or Red/System code. For example:
```

Comment:
"There is an exception for described naming conventions, which applies only to OS imports and third-party libraries not written in Red: to make API-specific functions and structure fields easy to recognize, their original names should be used. This helps to visually distinguish imported names from regular Red or Red/System code".